devel/docker-compose.yml

```diff
@@ -26,14 +26,10 @@ services:
     volumes:
       - ${CERT_DIR}/ca-crt.pem:/etc/osbuild-composer/ca-crt.pem:z
       - ${WORKER_CONFIG_DIR}/osbuild-worker.toml:/etc/osbuild-worker/osbuild-worker.toml:z
-      - ${WORKER_CONFIG_DIR}/token:/etc/osbuild-worker/token:z
+      - ${WORKER_CONFIG_DIR}/secret:/etc/osbuild-worker/secret:z
     environment:
       - CACHE_DIRECTORY=/var/cache/osbuild-composer
     privileged: true
-    cap_add:
-      - MKNOD
-      - SYS_ADMIN
-      - NET_ADMIN
```

Comment:
Are these not needed to build an image?

Reply:
Ah yes, you're right. I was getting a weird error for a bit where `privileged` and `cap_add` were clashing. I'll fix that now